### PR TITLE
General update of the text.

### PR DESCRIPTION
**Pull request by jbodleyscott**

### CODE REVIEW

**Comment by thomascellerier** [comment]
Some of the content of the readme is autogenerated so from other parts of the repository so you might have to edit other files, I'll let @mandopaloooza comment on that though.

**Comment by jbodleyscott** [comment]
> Some of the content of the readme is autogenerated so from other parts of the repository so you might have to edit other files, I'll let @mandopaloooza comment on that though.

Sure - can do if I know what/where.

File: README.md [hunk 1]
```diff
@@ -134,6 +135,6 @@ When running on GKE, the firewall needs to be configured to allow traffic from t
 | `rbac.create`           | Whether to create & use RBAC resources or not        | `true`      |
 | `service.type`          | Type of the service                                  | `ClusterIP` |
 | `service.port`          | Port of the service                                  | `443`       |
-| `replicaCount`          | Number of SDP client replicas to deploy              | `1`         |
+| `replicaCount`          | Number of SDP Client replicas to deploy              | `1`         |
```
Comment (mandopaloooza) [inline README.md:138]: As Thomas mentioned, the Parameters section is autogenerated using https://github.com/bitnami-labs/readme-generator-for-helm

To fix this portion of the README, you need to edit [values.yaml line 92](https://github.com/appgate/sdp-k8s-client/blob/d311e837b65ddae7d796923004faf169be22f6c4/k8s/chart/values.yaml#L92) then run the README generator. 

If you could just edit the values.yaml then I can run the generator afterwards.

**Review by mandopaloooza — CHANGES_REQUESTED** [review]
@jbodleyscott Could you also please review https://github.com/appgate/sdp-k8s-client/blob/main/k8s/chart/templates/NOTES.txt? 

NOTES.txt gets printed on the command line after the user installs the helm chart.

File: README.md (hunk 1 above)
Comment (jbodleyscott) [reply README.md:138]: done - you  can do the generator bit pls.

**Comment by jbodleyscott** [comment]
Also updated text based on @thomascellerier 's comments about 'new' etc.

File: README.md (hunk 1 above)
Comment (mandopaloooza) [reply README.md:138]: I could add the commit to this PR but it's a fork in your github namespace so I can't access it.

You can either grant me access to this or I can push the generator bit afterwards in a different PR

**Pull request merged**